apps/eth_blockchain/lib/eth_blockchain/adapter.ex

```diff
@@ -163,6 +181,14 @@ defmodule EthBlockchain.Adapter do
   @spec call(call(), atom() | adapter() | nil, server()) :: resp({:ok, any()})
   def call(func_spec, adapter_spec \\ nil, pid \\ nil)
 
+  def call({:send, attrs}, adapter, pid) do
```

Comment:
Nope, because there is a bunch of business logic handled at the `EthBlockchain` level (handled in `EthBlockchain.Transaction` and therefore we can't call the Geth worker directly.